.gitignore

```diff
@@ -0,0 +1,13 @@
+**/*.log
+**/node_modules
+.coverage
+.nyc_output
+docs/
+out/
+build/
+system-test/secrets.js
+system-test/*key.json
+*.lock
+.DS_Store
+google-cloud-logging-winston-*.tgz
+google-cloud-logging-bunyan-*.tgz
```

Comment:
Not for this PR, but it would be great to add `.vscode` to the template for this